FABRIC-AND-CFT.md

```diff
@@ -4,85 +4,85 @@ This page highlights the main differences (both technical and philosophical) bet
    </td>
-   <td>Organizations interested in using opinionated, prebuilt Terraform modules.
+   <td>Organizations interested in using opinionated, prebuilt Terraform modules with a focus on Google Cloud best practices.
    </td>
   </tr>
   <tr>
    <td><strong>Configuration</strong>
    </td>
-   <td>Less opinionated allowing end users higher flexibility.
+   <td>Less opinionated, allowing end users higher flexibility and often offering more configuration options within a single module.
    </td>
-   <td>Opinionated by default, end users may need to fork if it does not meet their use case.
+   <td>Opinionated by default, end users may need to fork if it doesn't fully meet their use case.  Often requires composing multiple modules for complex configurations.
    </td>
   </tr>
   <tr>
    <td><strong>Extensibility</strong>
    </td>
-   <td>Built with extensibility in mind catering to fork and use patterns. Modules are often lightweight and easy to adopt / tailor to specific use cases.
+   <td>Built with extensibility in mind, catering to fork and use patterns. Modules are often lightweight and easy to adopt / tailor to specific use cases.
    </td>
-   <td>Not built with fork and use extensibility catering primarily to off the shelf consumption. Modules are tailored towards common usecases and extensible via composition.
+   <td>Not primarily built for fork-and-use extensibility, catering primarily to off-the-shelf consumption. Modules are tailored towards common use cases and extensible via composition.
    </td>
   </tr>
   <tr>
-   <td><strong>Config customization</strong>
+   <td><strong>Config Customization</strong>
    </td>
-   <td>Prefer customization using variables via objects, tight variable space.
+   <td>Prefer customization using variables via objects, providing a tight but comprehensive variable space within modules.
    </td>
-   <td>Prefer customization using variables via primitives.
+   <td>Prefer customization using variables via primitives, which can lead to more modularity but may require using multiple modules for customization.
    </td>
   </tr>
   <tr>
    <td><strong>Examples</strong>
    </td>
-   <td>Thorough examples for individual modules, and end to end examples composing multiple modules covering a wide variety of use cases from foundations to solutions.
+   <td>Thorough examples for individual modules and end-to-end examples composing multiple modules, covering a wide variety of use cases from foundations to solutions. 
    </td>
    <td>Examples for a module mostly focus on that individual module. Composition is often not shown in examples but in larger modules built using smaller modules.
    </td>
   </tr>
   <tr>
    <td><strong>Resources</strong>
    </td>
-   <td>Leaner modules wrapping resources.
+   <td>Modules often encapsulate a broader set of resources, providing more functionality within a single unit.
```

Comment:
"often" is ambiguous. Does it happen or not?

What are you trying to convey with this change?

Reply:
You're right, "often" is too vague. 

I was trying to highlight that Fabric modules sometimes handle a wider range of resources or functionality within a single module, compared to CFT's tendency to break things down into smaller, more focused units.

However, without concrete examples, this distinction becomes blurry.  

**Instead of a general statement, would it be more helpful to:**
1. **Remove the "Resources" row entirely:** If there's no clear and consistent difference in how Fabric and CFT handle resources *in general*, it might be best to remove this point of comparison. 
2. **Replace it with specific module comparisons:** We can use the Google Storage and Project modules as examples where Fabric consolidates more functionality. We could say something like:
> "Fabric modules, such as the `gcs` and `project` modules, often manage a broader set of resources within a single unit compared to CFT's more modular approach. For instance, Fabric's `gcs` module handles IAM, tag bindings, and lifecycle rules directly, while CFT might use separate modules or resources for those aspects."

What do you think would be the clearest way to represent this difference?